python/ml4ir/base/data/tfrecord_writer.py

```diff
@@ -59,17 +59,21 @@ def write_from_files(
     logger: Logger = None,
 ):
     """
-    Converts data from CSV files into tfrecord data.
-    Output data protobuf format -> train.SequenceExample
-
-    Args:
-        csv_files: list of csv file paths to read data from
-        tfrecord_file: tfrecord file path to write the output
-        feature_config: str path to YAML feature config or str YAML feature config
-        tfrecord_type: TFRecordTypeKey.EXAMPLE or TFRecordTypeKey.SEQUENCE_EXAMPLE
-        logger: logging object
-
-    NOTE: This method should be moved out of ml4ir and into the preprocessing pipeline
+    Converts data from CSV files into tfrecord files
```

Comment:
the help at the beginning is included in the documentation : is it intended ?
and the formatting makes it really hard to read

![image](https://user-images.githubusercontent.com/63343516/95364233-dda91e80-08d0-11eb-8f56-8108bba6b8c1.png)

Reply:
This looks weird. I will take a look.